app/styles/_variables.scss

```diff
@@ -477,6 +477,11 @@ $overlay-background-color: rgba(0, 0, 0, 0.4);
   --dialog-information-color: #{$blue-400};
   --dialog-error-color: #{$red};
 
+  /** Banner */
+  --banner-warning-background: #{$yellow-700};
+  --banner-warning-text-color: var(--text-color);
+  --banner-warning-link-color: #{$blue-700};
```

Comment:
For accessibility, we need:
link -> text = 3:1 
link/text -> background: 4.5:1 

Unfortunately, this is giving link:text of 1.5:1 for light mode. 

Here is a link checker tool that has inputs for the link, the text, and the background to make verification easier.
https://webaim.org/resources/linkcontrastchecker/

Feel free to fix as you see fit. When I was testing locally, I tried the hidden bidi characters styles (which have been modified recently for accessibility.)  These colors were inspired by https://primer.style/components/banner. ... which one could argue that maybe we could use there "Critical" banner styles which would be a redish color kind of like dark mode. https://primer.style/foundations/primitives/color has a `--bgColor-danger-muted` (for background) and `--bgColor-danger-emphasis` (for icon)

```
  /** Banner */
  --banner-warning-background: #{$yellow-100}; // copied from --file-warning-background-color
  --banner-warning-text-color: var(--text-color);
  --banner-warning-link-color: var(--link-button-color);
  
  /_update-available.css 
    &.priority {
      .octicon {
          color: var(--file-warning-color);
        }
    ...
  ```
![Showing using same styles as from bidi character warning](https://github.com/user-attachments/assets/a602a15c-d9b3-4b5e-8ea5-facdcee06d7e)